.github/workflows/lint-tests-release.yml

```diff
@@ -16,6 +16,9 @@ jobs:
       - uses: actions/checkout@v3
       - run: ./scripts/lint_allowed_geth_imports.sh
         shell: bash
+      - uses: actions/setup-go@v3
+        with:
+          go-version: "1.19"
```

Comment:
added to ensure the go-version is correct (eg, if a PR was open and needed to downgrade)